### ISSUE TITLE
[create-threlte] vite plugin mckert asks for sudo password on startup (on linux)

### ISSUE THREAD

**Issue by wmurphyrd** [threlte/threlte#1109]
Excited to try this library so I run an npm create and then npm run dev and I get this (Ubuntu 22.04):

![image](https://github.com/user-attachments/assets/9a73aacb-daf9-45fa-821f-a5dbd1732d38)

Quite the frightening thing to see right after installing a bunch of new dependencies!

If you replace vite-plugin-mkcert with @vitejs/plugin-basic-ssl, then it will not prompt for sudo password. Only downside is you do have to click through the self-signed certificate warning on the browser to view the dev site.

If you're open to it, I' be happy to make the PR

**Comment by rmayer-sst** [comment]
Agreed.  Dev environments should not require sudo.

**Comment by grischaerbe** [comment]
@wmurphyrd Yes, this was reported before and I agree, we should swap it out. Feel free to make a PR ❤️

**Comment by grischaerbe** [comment]
We decided We decided to deprecate `npm create threlte` and want to implement a scaffolding by making use of Svelte's CLI `npx sv`.
